web/services/getUserProjectCustomerReportJsonService.php

```diff
@@ -35,67 +35,62 @@
 
     $userId = $_GET['uid'];
 
-    $init = $_GET['init'];
+    $init = $_GET['init'] ?? "";
```

Comment:
Any reason why not using `NULL` instead of `""`? I think it makes more sense

Reply:
I keep it like that because, later in the code, it checks for "" and not null (`if ($init!="")...`. I only wanted to get rid of the warning but not change a lot of code.

This kind of check is repeated over and over in the web services code, if I recall correctly, the original intention was to check for an empty parameter like in `getUserProjectCustomerReportJsonService.php?init=&...`.

Reply:
Oh I see, that's fine then, thanks

Reply:
Indeed, I checked and if you pass an empty GET parameter its value will be set as an empty string. I think it's a reasonable default.